pubspec.yaml

```diff
@@ -10,6 +10,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
+  meta: ^1.7.0
```

Comment:
question: any reason why we are adding `meta` here?

Reply:
yeah i couldn't publish the package because it was missing:

![Screen Shot 2022-01-18 at 12 32 27 PM](https://user-images.githubusercontent.com/16585828/149988535-cab993e5-b951-4b57-9298-6b073395b2b9.png)

it's imported in some of the files, but it was never added to pubspec!